packages/eslint-plugin-lit-a11y/template-analyzer/util.js

```diff
@@ -224,6 +240,10 @@ function templateExpressionToHtml(node) {
       html += `{{${expr.value}}}`;
     } else if (isCallExpression(expr)) {
       html += `{{{${getIdentifierName(expr.callee)}(${getCallExprValue(expr)})}}}`;
+    } else if (isMemberExpressions(expr)) {
+      html += `{{}}`;
+    } else if (isConditionalExpression(expr)) {
+      html += `{{}}`;
```

Comment:
These two clear the type issues, but I'm not sure if there _should_ be something in the placeholders?